src/ucs/sys/sys.c

```diff
@@ -1541,3 +1515,37 @@ long ucs_sys_get_num_cpus()
 
     return num_cpus;
 }
+
+unsigned long ucs_sys_get_proc_create_time(const char *proc_stat_path)
+{
+    char stat[1024];
+    char *start_str;
+    ssize_t size;
+    unsigned long stime;
+    int res;
+
+    size = ucs_read_file_str(stat, sizeof(stat), 1, proc_stat_path);
+    if (size < 0) {
+        ucs_diag("failed to read %s %m", proc_stat_path);
+        return 0ul;
```

Comment:
then ucs_diag will be printed twice
maybe return ucs_status to indicate if read was success or not exits. and return file time as long* param.